Tests/Data/StrTest.php

```diff
@@ -49,8 +49,10 @@ public function testConvertCase()
         $this->assertSame('LOWERCASE IN string', Str::ConvertCase('lowercase in STRING', 'UTF-8'));
     }
 
-    public function testIsBase64()
+    public function testisBase64()
```

Comment:
This changes I think should be reverted because of camel case naming approach.

Reply:
Merge fail, will do.